backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/chip/internal/BpkChipImpl.kt

```diff
@@ -86,6 +87,7 @@ internal fun BpkChipImpl(
                     enabled = enabled,
                     interactionSource = interactionSource,
                     indication = bpkRipple(),
+                    role = Role.Button,
```

Comment:
this is a selectable, the role should be `Role.Checkbox `
![Screenshot 2025-05-09 at 09 25 12](https://github.com/user-attachments/assets/cd3e81c2-6774-49c6-8d55-f39d5bb1979a)